onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -60,7 +63,6 @@
 '|^test_operator_non_float_params_cpu.*'
 '|^test_operator_params_cpu.*'
 '|^test_operator_pow_cpu.*'
-'|^test_shrink_cpu.*'
```

Comment:
Shrink test can be removed from exclusion (Test data bug fixed: https://github.com/onnx/onnx/commit/80346bd748fc8e586c70e89c50befe7d7f8c6383)